docs/resources/task.md

```diff
@@ -122,6 +122,14 @@ for all supported clouds.
 In addition to generic images, it's possible to specify any machine image
 supported by the underlying cloud provider.
 
+Images should include, at least:
+
+- Linux
+- cloud-init
+- systemd
+- curl
+- python3
```

Comment:
er, what's Python for?

Reply:
Extracting `zip` files, ¬obviously, because `unzip` needs to be installed and we would be relying on the distribution's package manager.

https://github.com/iterative/terraform-provider-iterative/blob/57625b6db38238a18dd10797ac239710c5f94f8a/task/common/machine/script.go#L64

Reply:
uh, Python just for `zipfile` seems a bit overkill but probably for a separate issue. Also works on both py2 & py3.

e.g.:

```
unzip rclone-current-linux-amd64.zip ||  python -m zipfile -e rclone-current-linux-amd64.zip .
```

Reply:
Feel free to open that separate issue. I'm curious to see if there is a more portable method.

Reply:
Using `python3` because `python` is not installed by default on modern distributions. Looking at you, Ubuntu.

Reply:
Python is really ubiquitous, nearly as Perl, while `unzip` isn't.

Reply:
Although, I somewhat like the idea of providing multiple fallbacks.

Reply:
Or, better yet, integrate `rclone` into the agent and avoid installing it separately.

Reply:
indeed, prefix `which rclone || `...

Reply:
#286

Reply:
```suggestion
- unzip, python or python3
```

Reply:
```suggestion
- unzip, python3 or python
```